inc/config.class.php

```diff
@@ -2212,6 +2212,12 @@ static function detectRootDoc() {
                                    str_replace('\\', '/', $currentdir));
          chdir($currentdir);
          $globaldir  = Html::cleanParametersURL($_SERVER['REQUEST_URI']);
+
+         // cron exception (running in CLI. Expect front slash to be missing)
+         if (strpos($globaldir, 'front/cron.php') !== false) {
```

Comment:
On CLI context, the `$_SERVER['REQUEST_URI']` corresponds to the path used in command line, so it may be `front/cron.php`, `./front/cron.php`, `glpi/front/cron.php`, or any absolute or relative path used by user.

Also, we will have same kind of problem when using `bin/console`.

But, the real question is: is it possible to detect `$CFG_GLPI["root_doc"]` in CLI context ? The purpose of this piece of code, unless I am wrong, is to detect how URL relative to domain should be prefixed. For instance, when you call `http://domain/glpi/front/index.php`, `root_doc` will have the `glpi` value. But when you call `php /var/www/glpi/front/cron.php`, how can you determine which value to use using the given path ?

URL generated in CLI/cron context should be absolute (i.e. using `$CFG_GLPI['url_base']`). We could eventually try to detect `root_doc` using `$CFG_GLPI['url_base']` in CLI context, but not sure we can do more.